.github/workflows/stale.yml

```
@@ -0,0 +1,19 @@
+name: Mark stale pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v3.0.7
```

```suggestion
    - uses: actions/stale@v3
```

GitHub Actions are expected to follow SemVer and common practice is to specify the major version only.